db/seeds.rb

```diff
@@ -121,20 +222,20 @@
 
   chef_profile = ChefProfile.new user: user, years_exp: i
   chef_profile.save!
-  rand(4..6).times do
+  rand(1..3).times do
     course = Course.new name: "#{Faker::Food.dish} course",
                         description: Faker::Food.description, cuisine_type: CUISINES.sample,
                         chef_profile: chef_profile, duration: rand(1..3) * 3_600_000,
                         price: rand(1..5) * 1500,
-                        rating: rand(1.0..5.0).round(2)
+                        rating: rand(3.0..4.0).round(2)
     course.save!
   end
 end
 
 puts "📑 Seed bookings ..."
 all_user = User.all
 all_chef = ChefProfile.all
-30.times do
+15.times do
   user = all_user.sample
   chef = all_chef.sample
   chef = all_chef.sample while user == chef.user
```

Comment:
Can we have a more constant booking? I found out last night that it's quite hard to find a sample of booking even with 30 booking randomly assigned to different user. Sometimes I need to login to three different chef account before I can find a chef with 'awaiting confirmation' booking status because some of them only have 2 bookings in history and status is confirmed or declined.